jenkins-pipelines/byo-longevity.jenkinsfile

```diff
@@ -0,0 +1,6 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+byoLongevityPipeline( )
```

Comment:
why it is empty?

Reply:
Since all parameters are coming from the user input. the whole idea of those .jenkinsfile was to have the defaults for each longevity. which isn't need in this case